statics/js/components/graph-layout.js

```diff
@@ -915,6 +916,34 @@ TopologyGraphLayout.prototype = {
     this.update();
   },
 
+  collapseByLevel: function(level, collapse, groups) {
+    if (level == 0) {
+      var i;
```

Comment:
move var at the beginning of the function as you use it in if and else and var scope is function

Reply:
done